src/components/LayerTree/LayerTree.md.scss

```diff
@@ -12,9 +18,83 @@
     margin: auto;
   }
 
-  .tm-map {
-    width: 100%;
-    height: 100%;
-    flex-grow: 1;
+  .tm-layer-tree-arrow {
```

Comment:
Where does come from all these style ? They weren't there before?

Reply:
duplicated due to merge